app/controllers/strategies_controller.rb

```diff
@@ -52,7 +52,7 @@ def edit
       redirect_to_path(strategy_path(@strategy))
     end
     @viewers = current_user.allies_by_status(:accepted)
-    @categories = current_user.categories.order('created_at DESC')
+    @categories = current_user.categories.is_visible.or(Category.where(id: @strategy.category_ids)).order('created_at DESC')
```

Comment:
I found a bug:

1. Create a Strategy with a visible Category
2. Edit the Category to be not visible
3. Edit the Strategy and uncheck the Category and hit "Submit"
4. The Category is not removed (it should be removed)

Reply:
Hmm. I will have to recreate this to know the game plan. But it will be done.